legacy/helmcharts/postgres-dbaas/values.yaml

```
@@ -4,4 +4,7 @@
 
 environment: ""
 
-readReplicaHosts: ""
\ No newline at end of file
+readReplicaHosts: ""
+
+imageCache: ""
+image: uselagoon/php-8.0-cli
```

This image has only got another month left - the 8.2 should just slot in here

```suggestion
image: uselagoon/php-8.2-cli
```

---

Can we figure out a better image to use maybe? We have `amazeeio/alpine-mysql-client` for mysql.

Should we maybe piece together a `uselagoon/database-tools` image instead that combines mysql, postgres, and also mongo tools required for the backup purposes?

---

Also considering the last tag on the `amazeeio/alpine-mysql-client` is [4 years ago](https://hub.docker.com/r/amazeeio/alpine-mysql-client/tags) maybe we should get something we can maintain better?